SETUP/tests/jsTests/dp_proofTests.js

```diff
@@ -21,6 +21,33 @@ QUnit.module("dp_proof tests", function(hooks) {
         delete window.inFace;
     });
 
+    QUnit.test("submitForm prevents double submit", function (assert) {
+        const done = assert.async();
+        const form = document.createElement('form');
+        const submitButton = document.createElement('input');
+        let submitCount = 0;
+        submitButton.type = 'submit';
+        form.addEventListener('submit', (event) => {
+            event.preventDefault();
+            submitForm(event.target);
+            submitCount++;
+            return false;
+        });
+        form.appendChild(submitButton);
+        document.body.appendChild(form);
+        assert.false(submitButton.disabled, "button should be disabled");
```

Comment:
Should this message say "button should be enabled"?

Reply:
yup, that is a copy paste error